pages/org-info.html

```diff
@@ -0,0 +1,140 @@
+        </div>
+        <div class="main__box">
+            <h3 class="main__sub-heading">College-Based Organizations</h3>
+            <div class="main__wrapper">
+                <div class="organization">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">College of ICT Student Council</a>
+                </div>
+                <div class="organization">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">College of Education Student Council</a>
+                </div>
+                <div class="organization">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">SILAK Media</a>
+                </div>
+                <div class="organization bottom-left">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">ICON Publication</a>
+                </div>
+                <div class="organization bottom-right">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">College of Nursing Student Council</a>
+                </div>
+            </div>
+            <a class="link" href="">See the existing university colleges and organizations that operate under it.</a>
+        </div>
+
+        <div class="main__box">
+            <h3 class="main__sub-heading">University-Based Organizations</h3>
+            <div class="main__wrapper">
+                <div class="organization">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">Forum Dimensions</a>
+                </div>
+                <div class="organization">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">LINK.exe</a>
+                </div>
+                <div class="organization">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">CYB:ORG</a>
+                </div>
+                <div class="organization bottom-left">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">WVSU-University Student Alliance</a>
+                </div>
+                <div class="organization bottom-right">
+                    <img src="assets/logos/USC-logo.png" alt="" class="org-logo">
+                    <a class="org-link" href="">SAMASA-WVSU</a>
+                </div>
```

Comment:
We'll have a problem with `.bottom-left` and `.bottom-right` CSS classes later on if we add more organizations to the list. This should be changed in the future but not in this pull request. We'll take note.